users/views.py

```diff
@@ -89,6 +97,35 @@ class UserCreate(BSModalCreateView):
     template_name = "modals/create_and_update_user.html"
     form_class = CustomUserCreationForm
 
+    def _send_create_user_email(self, user: CustomUser, password: str) -> None:
```

Comment:
Please order methods like I showed you example in one of previous comments

Reply:
Done.